tests/validators/tests.py

```diff
@@ -205,6 +205,11 @@
 
     (URLValidator(EXTENDED_SCHEMES), 'file://localhost/path', None),
     (URLValidator(EXTENDED_SCHEMES), 'git://example.com/', None),
+    (URLValidator(EXTENDED_SCHEMES), 'ssh://example.com/', None),
+    (URLValidator(EXTENDED_SCHEMES), 'git+ssh://git@github.com/example/hg-git.git', None),
+
+    # Empty schemes should not be accepted
+    (URLValidator(schemes=['']), '://www.example.com', ValidationError),
```

Comment:
I didn't find a better way to test validation without scheme since there was a predefined list of schemes. Also rejecting URLs with explicit `schemes=['']` makes sense because URLs starting with `://` are invalid by definition and no (practical) software supports these type of URLs.